src/common/gaussian.c

```diff
@@ -94,6 +94,13 @@ size_t dt_gaussian_memory_use(const int width,    // width of input image
   return mem_use;
 }
 
+size_t dt_gaussian_memory_use_CPU(const int width,    // width of input image
```

Comment:
I think about in the CPU case you want to call this routine instead of duplicating the computation. This would make it clear that both are related.

Reply:
And for consistency with other parts of dt, I would use a _cl suffix for the OpenCL version and nothing for the CPU.